docs/using-ngrok-with/docker.mdx

```diff
@@ -7,6 +7,12 @@ description: Learn how to use ngrok with Docker to expose your local application
 import TabItem from "@theme/TabItem";
 import Tabs from "@theme/Tabs";
 
+import { YouTubeEmbed } from "@components/youtube-embed";
```

Comment:
Do we want this placed above the title?

<img width="1656" height="1197" alt="image" src="https://github.com/user-attachments/assets/bcdf4006-a350-4cef-b21a-7c7ecbcede2c" />